v23.1/show-jobs.md

```diff
@@ -33,6 +33,8 @@ To block a call to `SHOW JOBS` that returns after all specified job ID(s) have a
 
 By default, only the `root` user can execute `SHOW JOBS`.
 
```

Comment:
"For changefeeds, users with the [`CHANGEFEED`](create-changefeed.html#required-privileges) privilege on a set of tables can view changefeed jobs running on those tables"

This applies to PAUSE, CANCEL, and RESUME

Reply:
Done!